client-js/app/scripts/obligation.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Obligation is the input side complement of Promise; if one party is promised
+ * a value, then of necessity another party must be obliged to provide the
```

Comment:
Eh, that's a 60% / 40% split. I'd say that's easily within the range where we can pick whichever we like better. Besides, most of those hits are probably middle-schoolers' Facebook posts :P